cli/cli-client/src/main/java/org/infinispan/cli/commands/Inc.java

```diff
@@ -0,0 +1,36 @@
+package org.infinispan.cli.commands;
+
+import org.aesh.command.Command;
+import org.aesh.command.CommandDefinition;
+import org.aesh.command.CommandResult;
+import org.aesh.command.impl.completer.BooleanOptionCompleter;
+import org.aesh.command.option.Argument;
+import org.aesh.command.option.Option;
+import org.infinispan.cli.activators.ConnectionActivator;
+import org.infinispan.cli.completers.CounterCompleter;
+import org.infinispan.cli.impl.ContextAwareCommandInvocation;
+import org.kohsuke.MetaInfServices;
+
+/**
+ * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
+ * @since 10.0
+ **/
+@MetaInfServices(Command.class)
+@CommandDefinition(name = Inc.CMD, description = "Increments a counter", activator = ConnectionActivator.class, aliases = "increment")
```

Comment:
Ah, this is the increment by one command I was looking for before. It might be a bit cleaner to consolidate `inc` and `add` to one command and just have the value argument optional.